src/libpmem2/deep_flush_linux.c

```diff
@@ -30,23 +30,44 @@ pmem2_deep_flush_write(unsigned region_id)
 
 	char deep_flush_path[PATH_MAX];
 	int deep_flush_fd;
+	char rbuf[2];
+	int rv = 0;
 
 	if (util_snprintf(deep_flush_path, PATH_MAX,
 		"/sys/bus/nd/devices/region%u/deep_flush", region_id) < 0) {
 		ERR("!snprintf");
 		return PMEM2_E_ERRNO;
 	}
 
+	if ((deep_flush_fd = os_open(deep_flush_path, O_RDONLY)) < 0) {
+		LOG(1, "!os_open(\"%s\", O_RDONLY)", deep_flush_path);
+		return PMEM2_E_NOSUPP;
+	}
+
+	if (read(deep_flush_fd, rbuf, sizeof(rbuf)) != 2) {
+		LOG(1, "!read(%d)", deep_flush_fd);
+		rv = PMEM2_E_NOSUPP;
```

Comment:
If this file doesn't contain 2 characters this is a kernel bug. we need a test to detect this path(and fail on it) to detect kernel bugs (or just return